### DISCUSSION TITLE
Minimal question - Could still use Reflect.get(target, prop)

### DISCUSSION THREAD

**Discussion by ericzorn93** [pmndrs/valtio#309] [Ideas]
https://github.com/pmndrs/valtio/blob/5ed0c7e081c80be4d947deadf1ed45875ca06a49/src/vanilla.ts#L94

**Comment by dai-shi** [comment]
Yeah, I actually refactored in #298 to utilize `Reflect` everywhere.

**Reply by ericzorn93** [reply]
Awesome, great work on your state management library options!
